src/components/Button/Button.tsx

```diff
@@ -37,11 +38,13 @@ export const Button: React.FunctionComponent<ButtonProps> = (props) => {
     return React.cloneElement(<button {...(p as React.ButtonHTMLAttributes<HTMLButtonElement>)} />);
   };
 
-  if (rest.href !== undefined || link) {
-    return (
-      <ThemeConsumer>
-        {(theme) => {
-          const css = style(props, theme);
+  const theme = useTheme();
+  const css = style({ props, theme });
+
+  return (
+    <ThemeConsumer>
+      {(theme) => {
```

Comment:
We don't need `ThemeConsumer` as we are getting theme from `useTheme`.

Reply:
Resolved in 3522fcf824ee04e717c79a860bb0f9f0d11b8a4c